05-onchain.md

```diff
@@ -185,231 +161,192 @@ Similarly, when node A sees a *commitment transaction* from B:
+HTLC-spending transaction itself once they've extracted the payment
 preimage; the knowledge is not revocable.
 
-
-Note that in cases where both resolutions are possible (payment
-success seen after timeout, for example), either interpretation is
-acceptable; it is the responsibility of the other node spend it
+In cases where both resolutions are possible (when payment
+success is seen after timeout, for example), either interpretation is
+acceptable; it is the responsibility of the recipient to spend it
 before this occurs.
 
-
-If the commitment transaction is theirs, our signature alone is enough
+If the commitment transaction is the recipient's, the offerer's signature alone is enough
 to spend the HTLC output (see
-[BOLT #3](03-transactions.md#received-htlc-outputs)), but we need to
-do so, otherwise they could fulfill the HTLC after the timeout.  If
-the commitment transaction is ours, we need to use the HTLC-timeout
+[BOLT #3](03-transactions.md#received-htlc-outputs)), but the offerer needs to
+do so, otherwise the recipient could fulfill the HTLC after the timeout. If
+the commitment transaction is the offerer's, they need to use the HTLC-timeout
 transaction.
 
-
 The fulfillment of an on-chain HTLC delivers the `payment_preimage`
-required to fulfill the incoming HTLC (if it, too, is on-chain) or use
+required to fulfill the incoming HTLC (if it, too, is on-chain) or to use
 in the `update_fulfill_htlc` message for the incoming HTLC.
-Otherwise, it needs to send the `update_fail_htlc` (presumably with
+Otherwise, the node needs to send the `update_fail_htlc` (presumably with
 reason `permanent_channel_failure`) as detailed in [BOLT
 02](https://github.com/lightningnetwork/lightning-rfc/blob/master/02-peer-protocol.md#forwarding-htlcs).
 
-
 If an HTLC is too small to appear in *any* commitment transaction, it
 can be safely failed immediately and `update_fail_htlc` returned to
-the incoming HTLC (if any: it might be locally-generated).  Otherwise,
-if a HTLC isn't in the commitment transaction we need to make sure
+the incoming HTLC (if any: it might be locally-generated).
+
+If a HTLC isn't in the commitment transaction a node needs to make sure
 that a blockchain reorganization or race does not switch to a
-commitment transaction which does contain it, before we fail it, hence
-the wait.  The requirement that we fail the incoming HTLC before its
+commitment transaction that does contain it before the node fails it, hence
+the wait. The requirement that the incoming HTLC be failed before its
 own timeout still applies as an upper bound.
 
-
 # On-chain HTLC Output Handling: Their Offers
 
-
-Each HTLC output can only be spent by us if we have the payment
-preimage, or them if it has timed out.
+Each HTLC output can only be spent by us, as the recipient, if we have the payment
+preimage, or them, as the offerer, if it has timed out.
```

Comment:
The recipient can only spend its HTLC output if it possesses the payment preimage; while the offerer can only spend its HTLC output if the HTLC has timed out.

Reply:
I think it's important to keep the "us" and "them" terminology in there for now, as it purposefully links to the headers and general focus of these sections. (I'm adapting the "as the"->"the" as you did in previous section.)